src/IncrementalChecker.ts

```diff
@@ -359,4 +363,15 @@ export class IncrementalChecker implements IncrementalCheckerInterface {
       lints.map(this.createNormalizedMessageFromRuleFailure)
     );
   }
+
+  private emitFiles() {
+    const program = this.program;
```

Comment:
please use `this.program` instead of assigning it to `const program` - one variable less to track :)

Reply:
Done, thank you again Piotr. :)